ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -733,26 +733,29 @@
         [buttons/save {:on-click #(update! (conj (or data []) {}))
                        :label add-label
                        :label-style style-base/button-label-style
-                       :disabled (values/effectively-empty? (last data))}]])]))
+                       :disabled (if add-label-validate
+                                   (not (add-label-validate (last data)))
```

Comment:
I don't understand the name  `add-label-validate`, it doesn't add a label but determines disabled state

Reply:
add-label should be "add-new-row-button" or something.
Then add-label-validate would be "add-new-row-button-validate". But you wouldn't like that?
I will change it to :add-label-disabled?